### ISSUE TITLE
{accessibility} Change theme

### ISSUE THREAD

**Issue by asmaloney** [RocketChat/Rocket.Chat#27751]
### Search before asking

- [X] I had searched in the [issues](https://github.com/RocketChat/Rocket.Chat.Electron/issues?q=is%3Aissue) and found no similar issues.

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Operating System Version

12.6.2

### It happens on the web browser too?

Yes, it happens on the web browser too

### Rocket.Chat Desktop App Version

3.8.16

### Rocket.Chat Server Version

3.12.1

### Describe the bug

The server uses a dark theme which I cannot use (headaches, etc.).

There does not seem to be a way anywhere to change to a light theme. On the web I can _sort-of_ make it readable using a browser plugin, but that doesn't help when I want to use the application.

I've looked at all the settings available to me and can't find any way to switch to a light theme, which makes it unusable 😞 .

### How to Reproduce

Try to use a light theme on a server using a dark theme 😄 

### Describe your Expected behavior

I expect to be able to select a light theme in my preferences like I can with almost any other software. For example on Slack I can set a theme per workspace which is really handy.

### Are you willing to submit a code contribution?

- [ ] Yes, I am willing to submit a Pull Request!

**Comment by ameynaik-9** [comment]
Hello Sir I could be very grateful if you can assign me this issue

**Comment by Tushar98644** [comment]
is anyone working on the issue? if not i would like to implement it...

**Comment by ivans-netto** [comment]
@asmaloney Now Rocket.Chat has three themes available; light, light with high contrast and dark.

On your sidebar > open the user menu > accessibility and appearance. There you're going to find not only themes, but other accessibility features.
